src/App.css

```diff
@@ -1,38 +1,156 @@
+
+.clearfix:after {visibility: hidden; display: block; font-size: 0; content: ''; clear: both; height: 0; }
+
+html { -moz-box-sizing: border-box; -webkit-box-sizing: border-box; box-sizing: border-box;}
+
+*, *:before, *:after {box-sizing: inherit}
+
+.sr-only { position: absolute; width: 1px; height: 1px; margin: -1px; border: 0; padding: 0; white-space: nowrap; clip-path: inset(100%); clip: rect(0 0 0 0); overflow: hidden;}
+
+@font-face {
+  font-family: "Hind";
+ 
+  src: local("Hind-Regular"),
+    url("./fonts/Hind-Regular.ttf") format("truetype");
+}
+@font-face {
+  font-family: "Montserrat";
+
+  src: local("Montserrat"),
+    url("./fonts/Montserrat-VariableFont_wght.ttf") format("truetype");
+}
+
+
+html{
+  font-size: 62.5%;
+}
+
+body {
+  background-color: slateblue;
+  font-size: 2rem;
+}
+
+header {
+  text-align: center;
+}
+
+h1{
+  font-size: 6rem;
+  color: white;
+  font-family: Montserrat, sans-serif;
+}
+
+h2{
+  margin: 5px 1px 50px 1px;
+  font-family: Hind, sans-serif;
+  font-weight: 600;
+  font-size: 1.8rem;
+  color: darkslateblue;
+  text-transform: uppercase;
+}
+
+p {
+  font-size: 2rem;
+  font-family: Hind, sans-serif;
   text-align: center;
+  color: white;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+a {
+  text-decoration: none;
+  font-family: Hind, sans-serif;
+  font-size: 1.5rem;
+  font-weight: 600;
+  border: 2px solid darkslateblue;
+  padding: 2.5px 5px;
+  color: darkslateblue;
+  letter-spacing: -0.5px;
+  
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+a:hover{
+  background-color: darkslateblue;
+  color: white;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+.link{
+  text-transform: uppercase;
+  position: absolute;
+  bottom: 0;
+  margin-bottom: 5px;
+}
+
+ul{
+  background-color: white;
+  padding: 0;
   display: flex;
-  flex-direction: column;
-  align-items: center;
+  flex-wrap: wrap;
   justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
 }
 
-.App-link {
-  color: #61dafb;
+li{
+  width: calc((100%/3) - 220px);
+  margin: 30px;
+  padding: 25px 2%;
+  min-width: 220px;
+  max-width: 310px;
+}
+
+.itemContainer{
+  position: relative;
+  list-style: none;
+  margin: 20px;
+  padding: 0;
+  box-shadow: 5px 10px 15px grey;
+}
+
+.itemContainer:hover{
+  box-shadow:5px 10px 15px slateblue ;
+}
```

Comment:
Not a big deal (feel free to ignore this comment, haha), but I noticed that the entire itemContainer has a hover effect, but only "View recipe" is clickable, which some users might find unexpected (i.e. they might expect the whole container to be clickable, since it reacts to the cursor). Just a design detail to think about.

Reply:
I am going to have the pic have additional info on it when you hover over it but ran out of time... 🙄